src/EnergyPlus/WaterToAirHeatPumpSimple.cc

```diff
@@ -1145,29 +1145,23 @@ namespace WaterToAirHeatPumpSimple {
-                state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).RatedAirVolFlowRate *
-                PsyRhoAirFnPbTdbW(state,
-                                  state.dataEnvrn->StdBaroPress,
-                                  state.dataLoopNodes->Node(AirInletNode).Temp,
-                                  state.dataLoopNodes->Node(AirInletNode).HumRat,
-                                  RoutineName);
+            state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).AirMassFlowRate = state.dataLoopNodes->Node(AirInletNode).MassFlowRate;
             // If air flow is less than 25% rated flow. Then set air flow to the 25% of rated conditions
-            if (state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).AirMassFlowRate <
-                0.25 * state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).RatedAirVolFlowRate *
-                    PsyRhoAirFnPbTdbW(state,
-                                      state.dataEnvrn->StdBaroPress,
-                                      state.dataLoopNodes->Node(AirInletNode).Temp,
-                                      state.dataLoopNodes->Node(AirInletNode).HumRat,
-                                      RoutineName)) {
-                state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).AirMassFlowRate =
-                    0.25 * state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).RatedAirVolFlowRate *
-                    PsyRhoAirFnPbTdbW(state,
-                                      state.dataEnvrn->StdBaroPress,
-                                      state.dataLoopNodes->Node(AirInletNode).Temp,
-                                      state.dataLoopNodes->Node(AirInletNode).HumRat,
-                                      RoutineName);
-            }
+            // if (state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).AirMassFlowRate <
+            //    0.25 * state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).RatedAirVolFlowRate *
+            //        PsyRhoAirFnPbTdbW(state,
+            //                          state.dataEnvrn->StdBaroPress,
+            //                          state.dataLoopNodes->Node(AirInletNode).Temp,
+            //                          state.dataLoopNodes->Node(AirInletNode).HumRat,
+            //                          RoutineName)) {
+            //   state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).AirMassFlowRate =
+            //        0.25 * state.dataWaterToAirHeatPumpSimple->SimpleWatertoAirHP(HPNum).RatedAirVolFlowRate *
+            //        PsyRhoAirFnPbTdbW(state,
+            //                          state.dataEnvrn->StdBaroPress,
+            //                          state.dataLoopNodes->Node(AirInletNode).Temp,
+            //                          state.dataLoopNodes->Node(AirInletNode).HumRat,
+            //                          RoutineName);
+            //}
```

Comment:
I never did like this check on air flow I am glad you got rid of it. But for DX systems you can freeze the coil if air flow is too low. I think this is why this code was used. Since this is no longer used it would be good to warn (fatal? @mjwitte) the user if air flow is below 25% of rated air flow. For cont fan this means inlet air flow, for cyc fan this means inlet air flow / PLR. This could be a 1-time check in init since parent air flow is constant throughout the simulation and only changes between cont fan and cyc fan during the simulation.

Reply:
So, current code just silently changes the airflow rate to a minimum of 25% of rated during the simulation?  Sure, a warning is fine,  just be sure to add details about the changes to the PR description.

Reply:
@rraustad @mjwitte It's mostly correct that current code silently changes airflow rate to 25% of rated, but not only that. The key thing here is that it only changes airflow rate variables in COIL data struct, it didn't really change the system flow rate nor node flow rate. Therefore, the problem here causes the coil to operate at a different airflow rate (not only because of the bump-up to 25% but also because current code always initiates to rated air flow for full load calc, which is the bugfix here), thus calculates a larger PLR, delivered energy, electric energy etc.